### DISCUSSION TITLE
KirikauKiwi's Flow Bench Build

### DISCUSSION THREAD

**Discussion by KirikauKiwi** [DIY-Flow-Bench/DIY-Flow-Bench#183] [Build Threads]
I have been following the DIYFB project for a while and had most of the sensors and ESP32 available from other projects along with the requirement of a decent vacuum source required for another project, I have started on my own build of the DIYFB.

The Flow source is based on a Universal 2 Stage 1200w Bypass Vacuum Cleaner Motor and the Electronics side from the M5 Stack lineup.

<img width="436" alt="Screenshot 2024-10-28 at 12 03 00" src="https://github.com/user-attachments/assets/3c14b359-4ce3-466d-b2f3-f21f4930ec36">

The plan is to 3D print the manifolds for the vacuum motors and set it up as a MAF style bench.

The parts so far are;
Paxanpax [Universal 2 Stage 1200w Bypass Motor 42-VM-998/PCM012](https://www.amazon.co.uk/dp/B01FFP8XTE?)
- 1340W measured
- 1389 VA
- PF 0.95
- 5.75 Amps
- 240 Volts
- RPM ????

M5 Stack [Core 2](https://shop.m5stack.com/products/m5stack-core2-esp32-iot-development-kit-v1-1) ESP32 Device.
M5 Stack [Env Pro](https://shop.m5stack.com/products/env-pro-unit-with-temperature-humidity-pressure-and-gas-sensor-bme688) (BME688) for Air: Pressure, Temp and RH).
M5 Stack [ADC](https://shop.m5stack.com/products/adc-i2c-unit-v1-1-ads1100) for reading the MAF sensor.
M5 Stack [Tube Pressure](https://shop.m5stack.com/products/tube-air-pressure-unit) -100 to 200 kPA for PRef,
M5 Stack [Pb.Hub](https://shop.m5stack.com/products/i-o-hub-1-to-6-expansion-unit-stm32f0) for ADC of TubePressure and also IO expansion PWM etc if needed.
Kemo Power Control [M280N](https://www.kemo-electronic.de/en/Light-Sound/Effects/Modules/M028N-Power-control-110-240-V-AC-4000-VA.php)
- 0-24VAC control input

Kemo DC + Pluse converter [M150](https://www.kemo-electronic.de/en/Transformer-Dimmer/Converter/M150-DC-pulse-converter.php)

![IMG_7865 Large](https://github.com/user-attachments/assets/91775e58-56b3-4a03-93a8-b97d6e73d27b)

**Comment by DeeEmm** [comment]
Looks like an interesting vac source. Any more details?
